### ISSUE TITLE
AdamW does not have args warmup and t_total

### ISSUE THREAD

**Issue by sophiapeng0426** [huggingface/transformers#830]
In finetune_on_pregenerated.py, below code throw error cause AdamW does not have those two arguments. This can be fixed by comment out those two columns but not sure if that means warmup will be not effective after that?

optimizer = AdamW(optimizer_grouped_parameters,
                             lr=args.learning_rate)
                            #warmup=args.warmup_proportion,
                           # t_total=num_train_optimization_steps)

**Comment by thomwolf** [comment]
Yes this example should have been updated now by #797.

Regarding `AdamW` and the schedule, details, and examples for the conversion are indicated in the migration section of the readme: https://github.com/huggingface/pytorch-transformers#Migrating-from-pytorch-pretrained-bert-to-pytorch-transformers
